scripts/copy_docs.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs/promises');
+const path = require('path')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+
+async function copyFiles (src, dest, version) {
+    const files = await fs.readdir(src, {withFileTypes: true})
+    for (const file of files) {
+        if (!file.name.startsWith('.')) {
+            if (file.isDirectory()) {
+                const newSrc = path.join(src,file.name)
+                const newDest = path.join(dest, file.name)
+                fs.mkdir(newDest, {recursive: true})
+                await copyFiles(newSrc, newDest, version)
+            } else {
+                const srcFile = path.join(src, file.name)
+                const destFile = path.join(dest, file.name)
+                if (!file.name.endsWith('.md')) {
+                    await fs.copyFile(srcFile, destFile)
+                } else {
+                const { stdout } = await exec(`git log -1 --pretty=format:%ci ${file.name}`, {
+                    cwd: src 
+                })
+                const header = '---\n' +
+                `originPath: ${path.join(src, file.name)}\n` +
+                `updated: ${stdout}\n` +
+                `version: ${version}\n` +
+                '---\n'
+                const content = await fs.readFile(srcFile)
```

Comment:
This doesn't include the changes from https://github.com/flowforge/website/pull/304/files#diff-87fcd60654beba05681c9de3d7c5fcf12e464f4396566688b33f4ecf48ffc45cR34

https://github.com/flowforge/website/blob/05b4534c93acd3f92fd10e36cf43660bdeb39523/scripts/copy_handbook.sh#L34

Reply:
Same for the handbook!

Reply:
The sed line is not in the original `copy_docs.sh`, but there is an equivalent in the `copy_handbook.js` on line 29

```
const body = header + content.toString().replace('---\n---\n', '')
```

I think I've understood what the sed line does and this should be the same because it's acting on the same file.